### DISCUSSION TITLE
Import Balance History from Mint?

### DISCUSSION THREAD

**Discussion by SawkeeReemo** [firefly-iii/firefly-iii#8302] [Firefly III]
I'm trying to migrate from Mint.com to Firefly, but I'm struggling to understand how to set things up a bit in a few places. First of all balance for bank accounts... According to the docs, if you set an opening balance, then any transactions that get imported will mess up the balance total. ...so is there a way to import a balance history? Or how do we fix this so balances are properly calculated? How do you set a current balance? Or don't you? I'm so confused. 😂 

I'm not an accountant, so some of these terms are completely foreign to me. My apologies for any newbness here.

**Comment by JC5** [comment]
A balance history and a transaction history are the same thing. Transactions change the balance.

What you do is, ignore the balance until you've imported your data. Then, calculate the difference:

- Expected balance: 123.45
- Firefly III says it's: 456.45
- So your opening balance should be -333 (123.45-456.45)

Or:

- Expected balance: 123,45
- Firefly III says it's: 40.23
- Your opening balance should be 83.22
